### PR TITLE
Add amazonlambda-connector-example.md

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+ adding: addingNumbers.py.py (deflated 17%)
+```
+## Upload Amazon Lambda Deployment Package (ZIP archive) in to the AWS S3 bucket
+1. Log in to the AWS Management Console.
+2. Navigate in to the created S3 bucket (Ex: eiconnectortest).
+3. Click Upload button.
+4. Select created Amazon Lambda Deployment Package (ZIP archive) and Upload.
+
+## Create Execution Role
+1. You need to create an Execution Role using [Amazon Lambda Connector Configuration](../amazonlambda-configuration.md) documentation.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
You need to create an Execution Role using [Amazon Lambda Connector Configuration](../amazonlambda-configuration.md) documentation. 
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
+# Amazon Lambda Connector Example
+
+AWS Lambda is a compute service that lets you run code without provisioning or managing servers. AWS Lambda executes your code only when needed and scales automatically, from a few requests per day to thousands per second. 
+The following operations allow you to work with functions in Amazon Lambda.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Seems irrelevant since we are not listing operations here. Let's remove this line,

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+    <tags>{$ctx:tags}</tags>
+    <timeout>{$ctx:timeout}</timeout>
+    <mode>{$ctx:mode}</mode>
+    <securityGroupIds>{$ctx:securityGroupIds}</securityGroupIds>
+    <subnetIds>{$ctx:subnetIds}</subnetIds>
+</amazonlambda.createFunction>
+````
+The following diagram illustrates all the required functionality of the Amazone Lambda Service that you are going to build.
+
+<img src="/assets/img/connectors/AmazonLambdaConnectorSample.png" title="Amazon Lambda Connector" width="800" alt="Amazon Lambda Connector"/>
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Give some explanation of the diagram as well.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+    <role>{$ctx:role}</role>
+    <runtime>{$ctx:runtime}</runtime>
+    <tags>{$ctx:tags}</tags>
+    <timeout>{$ctx:timeout}</timeout>
+    <mode>{$ctx:mode}</mode>
+    <securityGroupIds>{$ctx:securityGroupIds}</securityGroupIds>
+    <subnetIds>{$ctx:subnetIds}</subnetIds>
+</amazonlambda.createFunction>
+````
+The following diagram illustrates all the required functionality of the Amazone Lambda Service that you are going to build.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
The following diagram illustrates all the required functionality of the Amazon Lambda Service that you are going to build.
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+    <secretAccessKey>{$ctx:secretAccessKey}</secretAccessKey>
+    <accessKeyId>{$ctx:accessKeyId}</accessKeyId>
+    <blocking>{$ctx:blocking}</blocking>
+</amazonlambda.init>
+````
+**createFunction**
+
+The createFunction operation creates a Lambda function. To create a function, you need a deployment package and an execution role. The deployment package contains your function code. The execution role grants the function permission to use AWS services.
+
+You can add the parameter as follows in the createFunction operation:
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
You can add the parameter as follows in the `createFunction` operation:
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+<amazonlambda.init>
+    <region>{$ctx:region}</region>
+    <secretAccessKey>{$ctx:secretAccessKey}</secretAccessKey>
+    <accessKeyId>{$ctx:accessKeyId}</accessKeyId>
+    <blocking>{$ctx:blocking}</blocking>
+</amazonlambda.init>
+````
+**createFunction**
+
+The createFunction operation creates a Lambda function. To create a function, you need a deployment package and an execution role. The deployment package contains your function code. The execution role grants the function permission to use AWS services.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
The `createFunction` operation creates a Lambda function. To create a function, you need a deployment package and an execution role. The deployment package contains your function code. The execution role grants the function permission to use AWS services.
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+    </api> 
+    ```
+        
+2. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Select **Run on Micro Integrator**.
+3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+{! /references/connectors/exporting-artifacts.md !}
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Please check format. This doesn't render.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 8]
```diff
@@ -0,0 +1,262 @@
...
+       value2 = event['key2']
+       print("value1 = " + value1)
+       print("value2 = " + value2)
+       return float(value1) + float(value2) # Echo back the addition of two keys
+       #raise Exception('Something went wrong')
+```
+
+2. Create a ZIP archive.
+
+Please use command line terminal or shell to run following commands.Commands are shown in listings preceded by a prompt symbol ($) and the name of the current directory, when appropriate:
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
Please use command line terminal or shell to run following commands. Commands are shown in listings preceded by a prompt symbol ($) and the name of the current directory, when appropriate:
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 9]
```diff
@@ -0,0 +1,262 @@
...
+```
+~/Documents$ zip addingNumbers.zip addingNumbers.py
+ adding: addingNumbers.py.py (deflated 17%)
+```
+## Upload Amazon Lambda Deployment Package (ZIP archive) in to the AWS S3 bucket
+1. Log in to the AWS Management Console.
+2. Navigate in to the created S3 bucket (Ex: eiconnectortest).
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
2. Navigate in to the created S3 bucket (e.g., eiconnectortest).
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 10]
```diff
@@ -0,0 +1,262 @@
...
+3. Click Upload button.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
3. Click **Upload**.
```

Use bold for buttons, links, and other such items in the UI.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 11]
```diff
@@ -0,0 +1,262 @@
...
+## Create Execution Role
+1. You need to create an Execution Role using [Amazon Lambda Connector Configuration](../amazonlambda-configuration.md) documentation. 
+
+## Deployment
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+
+## Testing
+1. Login in to the Micro Integrator CLI tool.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
1. Log in to the Micro Integrator CLI tool.
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 12]
```diff
@@ -0,0 +1,262 @@
...
+2. Provide default credentials admin for both username and password.
+
+3. In order to view the proxy services deployed, execute the following command.
+
+    ```
+    ./mi api show
+    ```
+4. Send a POST request using a CURL command or sample client.
+  ```
+curl -v POST -d '{
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Fix formatting. This line should be in line with the start of the three backticks.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 13]
```diff
@@ -0,0 +1,262 @@
...
+    "functionName":"eiLambdaConnector",
+    "handler":"addingNumbers.addingNumbers",
+    "role":"arn:aws:iam::610968236798:role/EIConnectorTestRole",
+    "runtime":"python3.7",
+    "apiVersionCreateFunction":"2015-03-31"
+}
+' "http://localhost:8290/createFunction" -H "Content-Type:application/json"  
+ ```
+5. See the following message content.
+ ```
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Fix formatting. This needs to be tabbed.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 14]
```diff
@@ -0,0 +1,262 @@
...
+        "StateReasonCode": null,
+        "LastUpdateStatusReasonCode": null
+    }
+```
+6. Log in to the AWS Management Console.
+
+7. Navigate in to the AWS Lambda and Functions tab.
+   <img src="/assets/img/connectors/AWSLambdaFunction.png" title="Amazon Lambda Function" width="800" alt="Amazon Lambda Function"/>
+   
+8. You can simply execute function as follow.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
8. You can simply execute the function as follows.
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 15]
```diff
@@ -0,0 +1,262 @@
...
+   
+   Navigate in to the **Configure test events**
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
   Navigate to **Configure test events**
```

Shall we make all these steps. I.e., "9. Navigate to.."

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Removed.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 16]
```diff
@@ -0,0 +1,262 @@
+# Amazon Lambda Connector Example
+
+AWS Lambda is a compute service that lets you run code without provisioning or managing servers. AWS Lambda executes your code only when needed and scales automatically, from a few requests per day to thousands per second. 
+The following operations allow you to work with functions in Amazon Lambda.
+
+Given below is a sample scenario that demonstrates how to create a Amazon Lambda Function in the AWS Lambda Service using the WSO2 Amazone Lambda Connector.
```
[the remark this answers is not in the document]
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 6 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 7 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 8 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 9 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 10 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 11 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 12 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 13 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 14 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 15 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 5 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Added small description about the whole sample.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 17]
```diff
@@ -0,0 +1,266 @@
...
+    <mode>{$ctx:mode}</mode>
+    <securityGroupIds>{$ctx:securityGroupIds}</securityGroupIds>
+    <subnetIds>{$ctx:subnetIds}</subnetIds>
+</amazonlambda.createFunction>
+````
+The following diagram illustrates all the required functionality of the Amazon Lambda Service that you are going to build.
+
+<img src="/assets/img/connectors/AmazonLambdaConnectorSample.png" title="Amazon Lambda Connector" width="800" alt="Amazon Lambda Connector"/>
+
+This example demonstrates, how to create an Amazon Lambda function easily using the WSO2 Amazon Lambda Connector. Before creating Amazon Lambda function inside the AWS Lambda service you need to implement required deployment package (Zip Archive) locally.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
This example demonstrates, how to create an Amazon Lambda function easily using the WSO2 Amazon Lambda Connector. Before creating an Amazon Lambda function inside the AWS Lambda service, you need to implement the required deployment package (ZIP Archive) locally.
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 18]
```diff
@@ -0,0 +1,266 @@
...
+
+As a next step simply creates an AWS S3 bucket and the deployment package should be uploaded into that bucket. This sample API contains a service that can be invoked through an HTTP POST request. Once the service is invoked, it creates a Lambda function inside the AWS Lanmda service.The person who, invoking the created lambda function, able to run without provisioning or managing servers.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
As a next step, simply create an AWS S3 bucket and the deployment package should be uploaded into that bucket. This sample API contains a service that can be invoked through an HTTP POST request. Once the service is invoked, it creates a Lambda function inside the AWS Lambda service. When the created Lambda function is invoked, it is able to run without provisioning or managing servers.
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 19]
```diff
@@ -0,0 +1,266 @@
...
+    </api> 
+    ```
+        
+2. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Select **Run on Micro Integrator**.
+3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+    {!/references/connectors/exporting-artifacts.md!}
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
{!/references/connectors/exporting-artifacts.md!} 
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 20]
```diff
@@ -0,0 +1,266 @@
...
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+    {!/references/connectors/exporting-artifacts.md!} 
+
+## Create Amazon Lambda Deployment Package (Lambda function) 
+In this scenario we created sample AWS Deployment Package (Lambda function) in Python.
+
+1. Our sample Deployment Package would look similar to the following (source view : addingNumbers.py).
+```
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Tab this code bloack.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 21]
```diff
@@ -0,0 +1,266 @@
...
+ adding: addingNumbers.py.py (deflated 17%)
+```
+## Upload Amazon Lambda Deployment Package (ZIP archive) in to the AWS S3 bucket
+1. Log in to the AWS Management Console.
+2. Navigate to the created S3 bucket (e.g., eiconnectortest).
+3. Click **Upload**.
+4. Select created Amazon Lambda Deployment Package (ZIP archive) and Upload.
+
+## Create Execution Role
+You need to create an Execution Role using [Amazon Lambda Connector Configuration](../amazonlambda-configuration.md) documentation.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Rename this link to match new title.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 22]
```diff
@@ -0,0 +1,266 @@
...
+    }
+```
+6. Log in to the AWS Management Console.
+
+7. Navigate to the AWS Lambda and Functions tab.
+   <img src="/assets/img/connectors/AWSLambdaFunction.png" title="Amazon Lambda Function" width="800" alt="Amazon Lambda Function"/>
+   
+8. You can simply execute the function as follows.
+   
+   Navigate to **Configure test events**
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Can we make this step 9?

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 23]
```diff
@@ -0,0 +1,266 @@
...
+   <img src="/assets/img/connectors/configureTestEvent.png" title="Configure Test Event" width="800" alt="Configure Test Event"/>
+   
+   **Create Test Event**
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Step 10. Also better add how to create a Test Event.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 24]
```diff
@@ -0,0 +1,266 @@
...
+   <img src="/assets/img/connectors/createTestEvent.png" title="Create Test Event" width="800" alt="Create Test Event"/>
+   
+   **Execute created test event**
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Step 11. Add how to execute the test event.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 17 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 18 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 19 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 20 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 21 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 22 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 24 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Added.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 23 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Updated.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 25]
```diff
@@ -0,0 +1,266 @@
...
+    }
+```
+6. Log in to the AWS Management Console.
+
+7. Navigate to the AWS Lambda and Functions tab.
+   <img src="/assets/img/connectors/AWSLambdaFunction.png" title="Amazon Lambda Function" width="800" alt="Amazon Lambda Function"/>
+   
+8. You can simply execute the function as follows.
+   
+9. Navigate to **Configure test events**
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Merge these two steps.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 26]
```diff
@@ -0,0 +1,266 @@
...
+   <img src="/assets/img/connectors/configureTestEvent.png" title="Configure Test Event" width="800" alt="Configure Test Event"/>
+   
+10. Click **Create new test event**.
+   <img src="/assets/img/connectors/createTestEvent.png" title="Create Test Event" width="800" alt="Create Test Event"/>
+   
+11. Navigate and select created test event from the drop down top right corner. Click *Test* button and execute the test event.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
11. Navigate and select the created test event from the dropdown in the top right corner. Click the **Test** button and execute the test event.
```

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 25 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md (hunk 26 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.

File: en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md [hunk 27]
```diff
@@ -0,0 +1,265 @@
...
+        "StateReasonCode": null,
+        "LastUpdateStatusReasonCode": null
+    }
+```
+6. Log in to the AWS Management Console.
+
+7. Navigate to the AWS Lambda and Functions tab.
+   <img src="/assets/img/connectors/AWSLambdaFunction.png" title="Amazon Lambda Function" width="800" alt="Amazon Lambda Function"/>
+   
+8. You can simply execute the function as follows.Navigate to **Configure test events**
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]:
```suggestion
8. Next you need to execute the function. Navigate to **Configure test events**.
```

Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md]: Done.
